### PR TITLE
itemusage

### PR DESCRIPTION
**Pull request by Prakkkmak**

### CODE REVIEW

File: code/UI/Crosshair.cs [hunk 1]
```diff
@@ -10,6 +12,15 @@ public Crosshair()
             StyleSheet.Load("ui/crosshair.scss");
             selectCircle = AddChild<SelectCircle>();
         }
+        [Event("buildinput")]
+        public void ProcessClientInput(InputBuilder input)
+        {
+            if (Local.Pawn is not MainPlayer player)
+                return;
+
+            if (input.Pressed(InputButton.Attack2)) player.Holding.UseOn(player);
```
Comment (Nowp) [inline code/UI/Crosshair.cs:21]: It is not crosshair responsibility, we should do a full UI rework anyway

Comment (Prakkkmak) [reply code/UI/Crosshair.cs:21]: No need a rework ! 
We can make a command for use the holding ? It's better ?

Comment (Nowp) [reply code/UI/Crosshair.cs:21]: Yep a command would be fine but is it logical to have the "Use" part in the crosshair component ? For me, crosshair is not the component responsible for item usage

Comment (Prakkkmak) [reply code/UI/Crosshair.cs:21]: Yeah ok you convince me. Bu do we have to put the inputs outside ui ?

Comment (Prakkkmak) [reply code/UI/Crosshair.cs:21]: maybe we can make a inputHandler

Comment (Prakkkmak) [reply code/UI/Crosshair.cs:21]: Related to #219

**Review by Nowp — APPROVED** [review]

**Pull request merged**